src/main/java/seedu/address/logic/commands/findcommand/FindMentorCommand.java

```diff
@@ -26,41 +27,71 @@
             + "Example: " + COMMAND_WORD + " n/John Doe";
     public static final String MESSAGE_SUCCESS = "Successfully ran the find command.";
 
-    private String name;
-    private String email;
-    private String phone;
-    private String organization;
+    private String nameNorm;
+    private String emailNorm;
+    private String phoneNorm;
+    private String organizationNorm;
+    private String nameExclude;
+    private String emailExcude;
+    private String phoneExclude;
+    private String organizationExclude;
     private Predicate<Mentor> findPredicate;
 
     public FindMentorCommand(
-            Optional<String> name,
-            Optional<String> email,
-            Optional<String> phone,
-            Optional<String> organization
+            FindCommandUtilEnum type,
+            Optional<String> nameNorm,
```

Comment:
sorry can i check what is nameNorm? Does Norm stand for Normal, meaning that the field is present (and not for exclusion?). So this means that sth like nameNorm and nameExclude cannot both be present?

Reply:
Yea nameNorm is present. They can both be present because the positive and negative strings have separate `argMultimaps`